src/test.py

```diff
@@ -0,0 +1,14 @@
+import unittest
+
+from tests import test_e2e
+
+
+class SequentialTestLoader(unittest.TestLoader):
+    def getTestCaseNames(self, testCaseClass):
+        test_names = super().getTestCaseNames(testCaseClass)
+        testcase_methods = list(testCaseClass.__dict__.keys())
+        test_names.sort(key=testcase_methods.index)
+        return test_names
+
+
+unittest.main(test_e2e, testLoader=SequentialTestLoader())
```

Comment:
I think following is better:
```
if __name__ == "__main__":
    unittest.main(test_e2e, testLoader=SequentialTestLoader())
```

Reply:
Thanks. I added a line that you suggested.